src/components/Article.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Article.css';
+
+function Article({ title, date, text }) {
+  return (
+    <article className="article">
+      <h2 className="article-title">{title}</h2>
+      <hr />
```

Comment:
I understand why you add the title to `h2` but according to the description, you should add it in the `h1` tag. This also applies to adding content that is not described in the task. But I just leave it for mentor's review